src/js/background/backgroundLogic.js

```diff
@@ -3,6 +3,13 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 const DEFAULT_TAB = "about:newtab";
+/**
+ * Firefox does not yet have the `tabGroups` API, which exposes this constant
```

Comment:
FYI `tabGroups` API is coming in 139, see dependencies of: https://bugzilla.mozilla.org/show_bug.cgi?id=1940631